crates/vm-superio/src/rtc_pl031.rs

```diff
@@ -210,7 +210,7 @@ impl<EV: RtcEvents> Rtc<EV> {
             ris: state.ris,
             // A struct implementing RtcEvents for tracking the occurrence of
             // significant events.
-            events,
+            events
```

Comment:
nit: this should not be part of the commit.

Reply:
removed it.